cmd/start_server/main.go

```diff
@@ -15,9 +15,28 @@
 package main
 
 import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/logger"
 	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/manager"
 )
 
 func main() {
-	manager.StartServer("localhost", "8000")
+	port := flag.String("port", "8000", "port to run gRPC server on")
+	logsProject := flag.String("logs-project", "", "GCP Project ID for where to store logs")
```

Comment:
Nit: Should this be "GCP Project ID **in which** to store logs"?

Reply:
Change flag to project-id, and changed variable name to projectID